apps/consumer-client/package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "dev": "ts-node build.ts dev",
-    "build": "tsc --noEmit && ts-node build.ts build",
+    "build": "NODE_ENV=production tsc --noEmit && ts-node build.ts build",
```

Comment:
maybe we should have a separate `build:production` script for this?

Reply:
Yes makes sense!